migrator/migrations/m_200_to_m_201_compliance_v2_for_4_5/migration_impl.go

```diff
@@ -0,0 +1,303 @@
+package m200tom201
```

Comment:
AFAIK this migraiton only migrates fields from the serialized proto to a new column. WDYTH about skipping the migration and let the sync between Sensor and Central handle it? Theoretically it should be eventual consistent and after a Central restart have the correct data.

Reply:
I thought about that.  My concern was what if we started deduping those or not sending them.  I can certainly do a test.  If that turns out to work, and we decide on that path then the only part of this we would need is my change to the proto for `ComplianceOperatorScanV2` as my previous change was not wise.

Reply:
Upon further review, the deduping logic prevents us from guaranteeing the objects will be re-written on a sync.